back/db/database_v1.sql

```diff
@@ -1,19 +1,17 @@
+#-----------------------------------------------------------
+#        GREATEST DATABASE IN THE UNIVERSE 
 #------------------------------------------------------------
-#        Script MySQL.
-#------------------------------------------------------------
-
 
+CREATE DATABASE cacompilepas;
 #------------------------------------------------------------
 # Table: User
 #------------------------------------------------------------
 
-CREATE TABLE User(
+CREATE TABLE caCompilePas.CCP_User(
```

Comment:
It's not snake_case, snake case is cacompilepas.course_sheet, or cacompilepas.is_admin